README.md

```diff
@@ -144,7 +144,7 @@ See the [ARIA-tools-docs repository](https://github.com/aria-tools/ARIA-tools-do
 
 ------
 ## Citation
-D. Bekaert, M. Karim, L. Justin, H. Hua, P. Agram, S. Owen, G. Manipon, N. Malarout, M. Lucas, G. Sacco, L. Pan, S. Sangha, and ARIA team (2019), *Development and Dissemination of Standardized Geodetic Products by the Advanced Rapid Imaging and Analysis (ARIA) Center for Natural Hazards*, The International Union of Geodesy and Geophysics (IUGG), Montreal
+Buzzanga, B., Bekaert, D. P. S., Hamlington, B. D., & Sangha, S. S. (2020). Towards Sustained Monitoring of Subsidence at the Coast using InSAR and GPS: An Application in Hampton Roads, Virginia. Geophysical Research Letters, 47, e2020GL090013. https://doi.org/10.1029/2020GL090013 
```

Comment:
Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6171457)

Reply:
Congratulations on the GRL publication @bbuzz31. Looks like a record short time between submission and acceptance, so it must have been written very well before you submitted it.

Reply:
Thanks very much @EJFielding!

Reply:
@bbuzz31 please fix codacy issue. Links should be provided as: [](). 

Please also back issue the reference change to the earlier version branches.